pegasus/forms/hoc_census.rb

```diff
@@ -3,10 +3,26 @@
 class HocCensus2017 < HocSignup2017
   def self.normalize(data)
     result = super
-    # Once the change to split hoc signup and hoc census has been deployed we
-    # can move the census specific data extraction here. For now we want the
-    # old signup form to continue recording census data and we don't want to
-    # duplicate the data extraction here.
+
```

Comment:
I might be missing something, but don't we also want to store information in hoc_census about the nces id or school name. If hoc_census doesn't collect those fields, how will we link the answers to the census questions back to their school?

Reply:
Since this handler inherits from the HocSignup2017 handler it gets all the from processing from there. We populate the census form with all of the fields from the signup form. This just needs to handle to additional, census specific data.

Reply:
Yes, makes sense. Thanks for clarifying!